modules/apps/headless/headless-discovery/headless-discovery-impl/src/main/java/com/liferay/headless/discovery/internal/jaxrs/resource/v1_0/HeadlessDiscoveryOpenAPIResourceImpl.java

```diff
@@ -255,7 +255,7 @@ private Map<String, List<String>> _getOpenAPIMap(String accept) {
 
 		String serverURL =
 			_portal.getPortalURL(_httpServletRequest) + _portal.getPathProxy() +
-				Portal.PATH_MODULE;
+				_portal.getPathContext() + Portal.PATH_MODULE;
```

Comment:
I think the `_portal.getPathProxy()` should be removed as the `pathContext()` method is already having into account the path proxy.

![image](https://user-images.githubusercontent.com/32699538/220905725-bd2108a2-20e1-4d4b-aa91-a30cbb7c648f.png)

![image](https://user-images.githubusercontent.com/32699538/220905857-4efe2dd8-7c19-4a1a-9d4e-e0f0fa7a78ca.png)